OpenRA.Mods.Common/OpenRA.Mods.Common.csproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <Platform Condition=" '$(Platform)' == '' ">x86</Platform>
```

Comment:
Please drop this and the formatting changes below

Reply:
Ugh, this seems to be something that mono does on it's own sometimes. Very annoying.

Reply:
Fixed.